src/components/TextEditor/TextEditor.css.ts

```diff
@@ -20,3 +28,57 @@ globalStyle(`${iconButton} svg`, {
   width: '15px',
   height: '15px',
 });
+
+export const codeStyle = style({
+  backgroundColor: colors.gray3,
+});
+
+export const tooltipButton = style({
+  display: 'flex',
+  padding: '6px',
+});
```

Comment:
```suggestion
export const tooltipButton = style({
  display: 'flex',
  padding: '6px',
  marginLeft: 'auto',
});
```

이미지는 눌리는 요소가 아니라 색을 다르게 주신 것 같은데 위치도 아예 달리 주는 것은 어떤가요?

Reply:
좋습니다~